app/services/coronavirus/pages/draft_updater.rb

```diff
@@ -27,7 +28,7 @@ def send
       page.update!(state: "draft")
     rescue GdsApi::HTTPErrorResponse => e
       error_handler(e, "Failed to update the draft content item. Try saving again.")
-    rescue DraftUpdaterError => e
+    rescue UpdateFailedError => e
```

Comment:
Should this error be re-raised so that forces any transactions in the controller to be rolled back?

Reply:
Will this error every be thrown?

Reply:
With this change it can't be caught as it was catching the error raised at line 22 which was DraftUpdaterError, however the types don't match. It seems a rather cryptic implementation.

I'd have thought you'd not need a catch here but to just let the exception raise.

As you're heading towards multiple exception types you'll want to have a think about what you view code to look like as otherwise you may be heading towards catching a bunch of different exception types which could be arduous.